src/components/results/candidate-results/personality-archetype/index.js

```diff
@@ -42,31 +42,29 @@ class PersonalityArchetype extends Component {
     );
 
     return (
-      <div>
+      <div className={style.container}>
         {perspective === "thirdPerson"
```

Comment:
The beginning of the ternary should be on the same like like:

`{perspective === "thirdPerson" ? (`